scripts/Interface/ui/drill/model/specifications.py

```diff
@@ -0,0 +1,210 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
At the end shall we rename this file to `configurations.py`?

Reply:
Indeed, this is much better. I changed it.